src/app/rpmostree-pkg-builtins.cxx

```diff
@@ -181,7 +181,11 @@ rpmostree_builtin_install (int argc, char **argv, RpmOstreeCommandInvocation *in
 
   auto is_ostree_container = CXX_TRY_VAL (is_ostree_container (), error);
   if (is_ostree_container)
-    return rpmostree_container_install_packages (argv, cancellable, error);
+    {
+      auto pkgs = rpmostree_strv_to_cxx_string_vec (argv);
```

Comment:
I don't care too much, but in this path in theory we could create `rust::Str`.  Actually though on second thought, let's only do memory optimizations in memory safe languages.